doc/guides/modules.rst

```diff
@@ -10,14 +10,342 @@ system those are called *modules*. These modules must be integrated with the
+  :ref:`modules_synchronization`
+* be aware of security vulnerability issues in the external project
+  and update the module repository to include security fixes, as
+  soon as the fixes are available in the upstream code base
+* list any known security vulnerability issues, present in the
+  module code base, in Zephyr release notes.
+
+
+  .. note::
+
+     Module code owners are not required to be Zephyr
+     :ref:`Maintainers <project_roles>`.
+
+**Merging:** The Zephyr Release Engineering team has the right and the
+responsibility to merge pull requests in the main branch of a module
+repository.
+
+  .. note::
+
+     The merging of pull requests in module repositories shall be coupled
+     with the corresponding manifest file updates in the Zephyr main tree.
+
+
+Maintaining module code base
+============================
+
+Updates in the Zephyr main tree, for example, in public Zephyr APIs,
+may require patching a module code base. The responsibility for keeping
+the module code base up to date is shared between the **contributor** of
+such updates in Zephyr and the module **code owner**. In particular:
+
+* the contributor of the original changes in Zephyr is required to submit
+  the corresponding patches that are required in module repositories, to
+  ensure that Zephyr CI on the pull request with the original changes, as
+  well as the module integration testing are successful.
+
+* the module code owner has the overall responsibility for maintaining
+  the module code base in-synch with the Zephyr main tree, by doing, for
+  example, occassionally, more advance testing of the module code base.
+  The code onwer is required to fix issues in the module code base that
+  have not been caught by Zephyr pull request CI runs.
+
+  .. note::
+
+    The above polices apply on all Zephyr modules. However, modules which
+    Zephyr depends on, for example vendor HALs, are more crucial to be kept
+    up-to-date with Zephyr main tree.
+
+
+
+Contributing changes to modules
+===============================
+
+Changes submitted to a module's code base should be limited to
+
+* patches required due to updates in the Zephyr main tree
+* urgent patches, for example those that fix security vulnerability issues,
+  and, therefore, should not wait to be fixed, first, in the external project.
+
+Non trivial changes to module code base, including changes in the module
+design or functionality should be discouraged, if the module has an upstream
+project repository. In that case, such changes shall be submitted to the
+upstream project, directly.
+
+:ref:`Submitting changes to modules <submitting_new_modules>` describes in
+detail the process of contributing changes to module repositories.
+
+Contribution guidelines
+-----------------------
+
+Contributing to Zephyr modules shall follow the generic project
+:ref:`Contribution guidelines <contribute_guidelines>`.
+
+**Pull Requests:** may be merged with minimum of 2 approvals, including
```

Comment:
This requirement is way too tough:
Here there is no need for an actual code review, the code has already been reviewed and merged upstream. If, in the worst case, a review would be needed, a maintainer approval would be more than enough.

Reply:
> This requirement is way too tough

I think that the requirement is general, as the paragraph is not only about already upstream reviewed changes, but also about changes that are `urgent patches, for example those that fix security vulnerability issues (...)`
And those may require more eyes.

Reply:
Discussed in the meeting, online. Having the same approval policy for PRs across the organization is favorable, in accordance with the voted rules, and increases quality, so we would rather keep this (for the benefits it brings) and try our best with reviewing code more efficiently.

Reply:
Marking this as resolved, since this has been discussed extensively.